cardano-cli/src/Cardano/CLI/EraBased/Run/Key.hs

```diff
@@ -417,8 +447,9 @@ convertBip32SigningKey signingKeyBs =
     Nothing ->
       Left $ CardanoAddressSigningKeyDeserialisationError signingKeyBs
 
--- | Read a file containing a Bech32-encoded Ed25519 BIP32 extended signing
--- key.
+{- | Read a file containing a Bech32-encoded Ed25519 BIP32 extended signing
```

Comment:
I think I got used to `haddock-style: single-line`, so I think I'd prefer it more. In the end I wouldn't mind much using either.